src/app/components/Spinner.jsx

```diff
@@ -0,0 +1,260 @@
+import Image from "next/image";
+
+export default function Spinner({
+  shiftRight,
+  shiftLeft,
+  dishes,
+  wheel,
+  price,
+  name,
+  description,
+  orderNow,
+  container,
+}) {
```

Comment:
I think this looks better if you use one parameter instead. This component receives a lot of options, so all those parameters don't look neat

```suggestion
export default function Spinner(props) {
```